src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -65,34 +68,44 @@ const useSaveAnonChatHook = () => {
               member,
               initAnonymousChat?.message?.cid
             );
-
             await userMember.saveOrUpdateIfExists(localDb);
           } catch (e) {
-            console.log('error saveChatFromOtherProfile userMember', e);
+            console.log(e);
           }
           try {
             const memberSchema = ChannelListMemberSchema.fromInitAnonymousChatAPI(
               initAnonymousChat?.message?.cid,
               uuid(),
               member
             );
-            memberSchema.saveIfNotExist(localDb);
+            await memberSchema.saveIfNotExist(localDb);
           } catch (e) {
-            console.log('error saveChatFromOtherProfile memberSchema', e);
+            console.log(e);
           }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, good job on adding error handling. But, as mentioned earlier, consider providing user feedback or implementing retry logic instead of just logging the error.



<!-- This is an auto-generated comment by OSS CodeRabbit -->